components/PowerInfoCard.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+import Image from 'next/image';
+
+type PowerInfo = {
+  src: string;
+  applicance: string;
+  kWh: string;
+};
+
+const PowerInfoCard: FC<PowerInfo> = ({ src, applicance, kWh }) => {
+  return (
+    <div className="w-full">
+      <div className="bg-ocf-gray-1000 pt-3 pb-8 rounded-lg flex justify-center align-middle">
```

Comment:
also I mentioned this in another comment, but having `rounded-lg` here is resulting in a rounded border still showing when transitioning between elements in the carousel. to remedy this, we can take `rounded-lg bg-ocf-gray-1000` out of the styles here and instead move them to the styles of a div wrapping the entire `Carousel` element